runhouse/servers/cluster_servlet.py

```diff
@@ -266,10 +283,22 @@ async def asend_status_info_to_den(self):
         # Delay the start of post_status_thread, so we'll finish the cluster startup properly
         await asyncio.sleep(STATUS_CHECK_DELAY)
         while True:
-            logger.info("Sending cluster status to Den.")
+            logger.info("Trying to send cluster status to Den.")
             try:
-                interval_size = DEFAULT_STATUS_CHECK_INTERVAL
+                is_config_updated = (
+                    await self.aupdate_status_check_interval_in_cluster_config()
+                )
+                interval_size = (await self.aget_cluster_config()).get(
+                    "status_check_interval", DEFAULT_STATUS_CHECK_INTERVAL
+                )
                 if interval_size == -1:
+                    if is_config_updated:
```

Comment:
maybe we should add a log here anytime the config is updated, not just when its turned off

Reply:
@jlewitt1 This log is added as part of `aupdate_status_check_interval_in_cluster_config` method